README.md

```diff
@@ -1,128 +1,189 @@
-# Rezolus
+# Rezolus: High-Resolution Systems Performance Telemetry
 
-Rezolus captures and exports high resolution data about systems performance. It
-sets itself apart from other telemetry agents by using:
+## What is Rezolus?
 
-* **eBPF**: Uses eBPF on Linux to instrument individual events and aggregate
-  them into distributions. Rezolus will report on things like block io size and
-  latency distributions, system call latency, TCP segment sizes, and more. By
-  using eBPF we are able to instrument low-level events efficiently and provide
-  new insights into system performance and workload characteristics.
+Rezolus is a Linux performance telemetry agent that provides detailed insights
```

Comment:
I wonder if we should emphasize more on `systems` than `performance` telemetry when it comes to Rezolus.